c2corg_ui/format/__init__.py

```diff
@@ -34,6 +35,7 @@ def _get_markdown_parser():
             C2CWarningExtension(),
             Nl2BrExtension(),
             TocExtension(marker='[toc]', baselevel=2),
+            AutoLinkExtension()
```

Comment:
No special reason I think, probably the unit tests have been forgotten when adding those extensions.